CeedlingExec.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+
+import sublime
+import sublime_plugin
+from Default.exec import ExecCommand
```

Comment:
I suggest importing it `as _ExecCommand` because otherwise ST will detect this import as a new `exec` command. Which doesn't matter in practice most of the time since it will still be the same underlying `exec`, but from a purist standpoint it would definitely be better.

Actually, you don't even need to subclass the `exec` command since you could as well just prepare the new args and then call it via `window.run_command`.

Reply:
Changed import as suggested.